src/CoreBundle/DataModel/BackendTreeNodeDataModel.php

```diff
@@ -24,6 +24,11 @@ class BackendTreeNodeDataModel implements JsonSerializable
      */
     private $name = '';
 
+    /**
+     * @var string
+     */
+    private $furtherIcons = '';
```

Comment:
Is this HTML?

Reply:
yes, itt is.. I renamed it to furtherIconsHTML